python/example/single_cell_recipe.py

```diff
@@ -1,13 +1,35 @@
+#!/usr/bin/env python3
+
 import arbor
-import numpy, pandas, seaborn # You may have to pip install these.
+import pandas, seaborn # You may have to pip install these.
 
 # The corresponding generic recipe version of `single_cell_model.py`.
 
-# (1) Define a recipe for a single cell and set of probes upon it.
+# (1) Create a morphology with a single (cylindrical) segment of length=diameter=6 μm
+tree = arbor.segment_tree()
+tree.append(arbor.mnpos, arbor.mpoint(-3, 0, 0, 3), arbor.mpoint(3, 0, 0, 3), tag=1)
+
+# (2) Define the soma and its center
+labels = arbor.label_dict({'soma':   '(tag 1)',
+                           'center': '(location 0 0.5)'})
+
```

Comment:
Is there a good reason to use 'center' over 'centre'? Because otherwise I'd like to stick to UK/EU spellings.

Reply:
I know you do ;) But this is what's we use in all other examples, even here (you wrote this line).

Reply:
Maybe we should change all of them!

Reply:
(And in this example, I did use the other spelling...)

Reply:
In this example, you used both spellings in fact, which was problematic ;)